mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.h

```diff
@@ -26,6 +26,18 @@ inline bool isSYCLOperation(Operation *op) {
   return isa<sycl::SYCLDialect>(op->getDialect());
 }
 
+inline unsigned targetToAddressSpace(Target target) {
+  switch (target) {
+  case Target::ConstantBuffer:
+  case Target::GlobalBuffer:
+    return 1;
+  case Target::Local:
+    return 3;
+  default:
+    llvm_unreachable("Invalid Target for an accessor");
+  }
+}
```

Comment:
`SYCLAttributes.h` might be a better place for this? We will ideally be performing an `attribute -> attribute` mapping here. Also, add a comment:
```
// TODO: Replace with TargetAttr -> AccessAddrSpaceAttr mapping
```

Reply:
done.